src/main/scala/cromwell/util/GcsUtil.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.util
+
+object GcsUtil {
```

Comment:
Oh - I didn't realize that `GoogleCloudStoragePath` was your own construct :) Just add this as a method directly.